inspirehep/modules/workflows/models.py

```diff
@@ -50,3 +58,16 @@ def save(self):
         """Save object to persistent storage."""
         with db.session.begin_nested():
             db.session.add(self)
+
+
+class WorkflowsPendingRecord(db.Model):
+
+    __tablename__ = "workflows_pending_record"
+
+    workflow_id = db.Column(db.Integer, primary_key=True, nullable=False)
```

Comment:
Can we do something similar with the record id? (if so, what should be the foreign key fqdn?)